services/sensu/config.go

```diff
@@ -11,6 +11,8 @@ type Config struct {
 	Addr string `toml:"addr" override:"addr"`
 	// The JIT sensu source name of the alert.
 	Source string `toml:"source" override:"source"`
+	// The sensu handler to use
+	Handlers []string `toml:"handlers"`
```

Comment:
Adding the `override` tag here will make it so that the tests above use the lower case name to be consistent.